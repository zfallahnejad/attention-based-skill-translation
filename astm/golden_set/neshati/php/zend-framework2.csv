958642,zend-framework2
105019,zend-framework2
1449814,zend-framework2
1112089,zend-framework2
434223,zend-framework2
1185624,zend-framework2
199593,zend-framework2
347063,zend-framework2
1055015,zend-framework2
1386551,zend-framework2
1331671,zend-framework2
605093,zend-framework2
23060,zend-framework2
412395,zend-framework2
172588,zend-framework2
1258925,zend-framework2
2061376,zend-framework2
1866605,zend-framework2
1022873,zend-framework2
1185180,zend-framework2
2031918,zend-framework2
1611613,zend-framework2
1493332,zend-framework2
290339,zend-framework2
1248820,zend-framework2
1105871,zend-framework2
1312094,zend-framework2
1742415,zend-framework2
2408594,zend-framework2
1935933,zend-framework2
4238918,zend-framework2
669073,zend-framework2
