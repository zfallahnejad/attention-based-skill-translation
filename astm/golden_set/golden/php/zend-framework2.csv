958642,97,71,0.5773809523809523
105019,64,41,0.6095238095238096
1449814,48,38,0.5581395348837209
1112089,42,14,0.75
434223,39,30,0.5652173913043478
1185624,30,37,0.44776119402985076
199593,26,32,0.4482758620689655
347063,23,8,0.7419354838709677
1055015,17,21,0.4473684210526316
1386551,16,8,0.6666666666666666
1331671,14,2,0.875
605093,13,11,0.5416666666666666
23060,12,12,0.5
412395,12,3,0.8
1258925,11,12,0.4782608695652174
1866605,11,10,0.5238095238095238
172588,11,6,0.6470588235294118
1022873,11,9,0.55
2061376,11,7,0.6111111111111112
1185180,10,13,0.43478260869565216
2031918,9,12,0.42857142857142855
1611613,9,3,0.75
2408594,8,2,0.8
1935933,8,6,0.5714285714285714
290339,8,11,0.42105263157894735
1493332,8,2,0.8
1312094,8,8,0.5
4238918,8,6,0.5714285714285714
1742415,8,6,0.5714285714285714
1105871,8,6,0.5714285714285714
1248820,8,6,0.5714285714285714
669073,7,4,0.6363636363636364
